README.md

```diff
@@ -431,6 +436,10 @@ content
 -------
 Content of the PAM file for the service
 
+lines
```

Comment:
Same here. This should also mention that it is mutually exclusive with content and that content or lines is required to be set if ensure is present.